pkg/trace/stats/aggregation.go

```diff
@@ -0,0 +1,91 @@
+package stats
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"strings"
+)
+
+const (
+	hostnameTag   = "_dd.hostname"
```

Comment:
Can we use the same style as in `http.StatusCode*` and other places where the prefix is the type? e.g.

```
tagHostname
tagStatusCode
tagVersion
```